Backend/register_user.php

```diff
@@ -0,0 +1,34 @@
+<?php
+require "init.php";
+
+//a ratings value could be set as default instead of null being used in db.
+$email= $_POST["loggedInUserEmail"];
+$lastName=$_POST["loggedInUserLastName"];
+$firstName=$_POST["loggedInUserFirstName"];
+$displayName=$_POST["loggedInUserDisplayName"];
+$deviceId=$_POST["loggedInUserDeviceId"];
+$profilPicUrl=$_POST["loggedInUserProfileUrl"];
+
+//Check if users exists, if not register user in database.
+$sql = "SELECT * FROM Users WHERE email LIKE '".$email."'";
+//echo $sql;
+$result = mysqli_query($con,$sql);
+//echo "\r\n check if exists";
+//var_dump($result);
+
+if(mysqli_num_rows($result) >= 1){
```

Comment:
There should be a space between the closing parenthesis and opening curly bracket.